examples/change_attribute/change_attitude.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
```

Comment:
This code clearly only runs with python2. I think it would be best to specify that, so nobody tries to run it and have it fail.